performance/SwiftUIPerformanceQuickstart/PerformanceExample/README.md

```diff
@@ -0,0 +1,54 @@
+Firebase Performance Monitoring Quickstart
+=============================
+
+# Firebase Performance Monitoring
+
+Firebase Performance Monitoring is a free mobile app performance analytics service. It
+provides detailed information about the performance of your apps automatically,
```

Comment:
provides details information about the performance of your apps (app start, network requests, screen performance) automatically.

Reply:
Done ✅